src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -2,64 +2,25 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
-using System;
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.FlowAnalysis;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Operations
 {
-    internal sealed class OperationCloner : OperationVisitor<object, IOperation>
+    internal sealed partial class OperationCloner : OperationVisitor<object?, IOperation>
+#nullable disable
```

Comment:
Slight worry this will make it appear to most of our tooling that this file has nullability enabled when in reality it doesn't. Consider leaving the `#nullable disable` at the top of the file and do a `enable / disable` pair here.

Reply:
It's only disabled for now because I haven't finished porting the rest of the types. The goal is for this file to either entirely disappear, or almost entirely disappear and be fully nullable-enabled, by the end.